### DISCUSSION TITLE
When a tiger attack occurs, the bots act like complete idiots.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13903] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When a tiger attack occurs, the bots act like complete idiots.
All my bots are armed with SMG or revolver
Once the tigers get inside,
The bots stop firing from their ship's cannons and start running around mindlessly, not shooting at the tigers with their hand guns.
Bots don't fire cannons at tigers that are outside.
Bots do not shoot from smgs and revolvers at those tigers that are inside.
Bots don't try to run away from tigers.
As a result, they quickly die.
I remember that in earlier versions of the game the bots behaved more intelligently. Seeing the enemy, they immediately began to shoot at him and usually killed the intruders before anyone died.
It looks like the bots were deliberately lowered in intelligence in the latest version or a little earlier.

### Reproduction steps

unzip the save

[20240424Camel.zip](https://github.com/FakeFishGames/Barotrauma/files/15156077/20240424Camel.zip)

move forward until you come across a pack of tigers
let the tigers get inside
order the bots to stand on the guns.
bots ignore guns and bots ignore tigers
bots run around in circles doing nothing sensible.
bots don't try to run away from the tiger or get treatment
What the hell is going on?

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by BlackMoons** [comment]
Can 100% confirm with following info:

a security bot on 'fight intruders' knows precisely where any enemy on the ship is, instantly, runs towards it, calls it out and then...

Every other bot abandons the turrets as soon as an AI calls out any enemy the see, letting the rest of the enemies onboard.

Bots then runs off to the enemy onboard and stands in a giant mob of AI's... Resulting in nobody firing their weapons due to high chance of friendly fire. 

And then everybody dies. Except maybe the very last guy since the friendly fire problem is fixed once all but one bot die.

Suggested simple solution (Bandaid tier): That bots *NOT* abandon their current task to join any AI who called out an enemy, only those on 'fight intruders' should respond to enemy calls outs. Other bots should maybe flee or fight if the intruder comes to them, but to NOT seek it out and join a giant ineffective mob while more enemies get on board. 

I repeat: all 10 bots on your ship running off to fight *1* intruder instead of fixing, shooting, repairing, healing, etc is completely ineffective and gets the entire crew killed as the other enemies easily board the ship due to complete lack of turret coverage.

Also, it would be nice if 'fight intruders' didn't mean 'instantly know where any intruder is, at all times' and instead actually listened to callouts for intruders (from players AND ai)
